### ISSUE TITLE
Use a different recipe for unit tests

### ISSUE THREAD

**Issue by yuvipanda** [pangeo-forge/pangeo-forge-runner#25]
Currently we use https://github.com/pangeo-forge/gpcp-feedstock/ for our unit tests, run with `--prune`. @rabernat suggested this might not be the best idea, and we should try something else. Opening this to track *if* we need to replace it, and if so, with what.

**Comment by cisaacstern** [comment]
Following merge of #26 (where tests passed), tests subsequently [failed with a 502](https://github.com/pangeo-forge/pangeo-forge-runner/actions/runs/3061814161/jobs/4942208637#step:7:132):

```
raise ClientResponseError(\\naiohttp.client_exceptions.ClientResponseError: 502, message=\'Proxy Error\', url=URL(\'[https://www.ncei.noaa.gov/data/global-precipitation-climatology-project-gpcp-daily/access/1996\](https://www.ncei.noaa.gov/data/global-precipitation-climatology-project-gpcp-daily/access/1996/)')", "status": "failed"}\n'
```
I believe this is related, insofar as the issue here is instability of the gpcp source server?

**Comment by cisaacstern** [comment]
Despite the fact that these ☝️ tests failed on merge, I still released `v0.5` to PyPI, which I believe is justified given that the unit tests passed on #26, and there were no code changes in between those passing tests and the failed tests on merge.
